assets/js/langpack/es.json

```diff
@@ -0,0 +1,126 @@
+{
+  "token": {
+    "Claim control of your":
+    "Obtén el control de tus",
+
+    "Docker images":
+    "Imágenes Docker",
+
+    "View on GitHub":
+    "Ver en GitHub",
+
+    "Download .zip":
+    "Descargar .zip",
+
+    "Download .tar.gz":
+    "Descargar .tar.gz",
+
+    "What is Portus":
+    "Qué es Portus",
+
+    "Portus is an open source authorization service and user interface for the next generation Docker Registry.":
+    "Portus es un servicio de autorización de libre código e interfaz de usuario para la próxima generación de Registros Docker",
```

Comment:
Portus is an open source ... I thought that translated into "codigo libre". You can change the strings as you please anyways ;)